### PR TITLE
fix: trim in dashboard port number of SNMP device address

### PR DESCRIPTION
**Pull request by ikheifets-splunk**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Dependency update
- [ ] Bug fix
- [ ] New feature
- [ ] Refactor/improvement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist

- [ ] My commit message is [conventional](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] I have run pre-commit on all files before creating the PR
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: dashboard/dashboard.xml [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
+          <query>index=* sourcetype="*:container:splunk-connect-for-snmp-*" "Scheduler: Sending due task sc4snmp;*;*;poll" | rex field=_raw "Sending due task sc4snmp;(?&lt;ip&gt;.+):(?&lt;port&gt;\d+);(?&lt;num&gt;\d+);poll" | stats count by ip</query>
           <earliest>-24h@h</earliest>
           <latest>now</latest>
         </search>
       </input>
       <chart>
         <title>In case of unsuccessful polling status, please copy spl query from this chart and find failed tasks. Explanation of error log messages you can find at the https://splunk.github.io/splunk-connect-for-snmp/main/bestpractices/</title>
         <search>
-          <query>index=* sourcetype="*:container:splunk-connect-for-snmp-*" splunk_connect_for_snmp.snmp.tasks.poll $poll_status_host$ | rex field=_raw "Task splunk_connect_for_snmp.*\[*\] (?&lt;status&gt;\w+)" | where status != "received" | timechart count by status</query>
+          <query>index=* sourcetype="*:container:splunk-connect-for-snmp-*" splunk_connect_for_snmp.snmp.tasks.poll "$poll_status_host$" | rex field=_raw "Task splunk_connect_for_snmp.*\[*\] (?&lt;status&gt;\w+)" | where status != "received" | timechart count by status</query>
```
Comment (ikheifets-splunk) [inline dashboard/dashboard.xml:23]: doublequoting address because IPv6 can consist escape symbols

File: docs/dashboard.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
 
 Using dashboard you can monitor SC4SNMP and be sure that is healthy and working correctly.
 
+
 ## Presetting
 
+!!! info
+    Dashboard compatible with SC4SNMP 1.12+
```
Comment (ikheifets-splunk) [inline docs/dashboard.md:9]: Add this disclaimer, because in 1.11 we not provided port number in IP

File: dashboard/dashboard.xml [hunk 3]
```diff
@@ -12,15 +12,15 @@
         <fieldForLabel>ip</fieldForLabel>
         <fieldForValue>ip</fieldForValue>
         <search>
-          <query>index=* sourcetype="*:container:splunk-connect-for-snmp-*" "Scheduler: Sending due task sc4snmp;*;*;poll" | rex field=_raw "Sending due task sc4snmp;(?&lt;ip&gt;.+);(?&lt;num&gt;\d+);poll" | stats count by ip</query>
+          <query>index=* sourcetype="*:container:splunk-connect-for-snmp-*" "Scheduler: Sending due task sc4snmp;*;*;poll" | rex field=_raw "Sending due task sc4snmp;(?&lt;ip&gt;.+):(?&lt;port&gt;\d+);(?&lt;num&gt;\d+);poll" | stats count by ip</query>
```
Comment (ikheifets-splunk) [inline dashboard/dashboard.xml:15]: it's escaping symbols, it's just `:(?<port>\d+)`

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,7 @@
 - add metrics dashboard
 - add support for IPv6 polling and traps for kubernetes deployment
 - add support for IPv6 polling and traps for docker deployment
+- add support new IP format in dashboard
```
Comment (ajasnosz) [inline CHANGELOG.md:9]: for

File: docs/dashboard.md (hunk 2 above)
Comment (ajasnosz) [reply docs/dashboard.md:9]: Is !!! some markdown? If not I think we can replace it with:
**Info
    Dashboard compatible with SC4SNMP 1.12+**

**Review by ajasnosz — CHANGES_REQUESTED** [review]

File: docs/dashboard.md (hunk 2 above)
Comment (ikheifets-splunk) [reply docs/dashboard.md:9]: Screenshot 2024-10-09 at 09 03 03

**Review by ajasnosz — APPROVED** [review]

**Pull request merged**